### ISSUE TITLE
json.h include path on (Arch) Linux

### ISSUE THREAD

**Issue by mstraube** [blockattack/blockattack-game#9]
On my Arch Linux system I have to change [line 29](https://github.com/blockattack/blockattack-game/blob/7c8588cdd96d5a1a28c17dbea56e250b8c4c70e9/source/code/sago/SagoSpriteHolder.cpp#L29) to `#ifdef _WIN32`

Otherwise the json.h header file is not found. 
```
blockattack-game-2.0.1/source/code/sago/SagoSpriteHolder.cpp:30:31: fatal error: jsoncpp/json/json.h: No such file or directory
 #include <jsoncpp/json/json.h>
                               ^
```
I'm not sure if this is a problem on other distributions, too.

The path on my system is `/usr/include/json/json.h`

**Comment by sago007** [comment]
Thank you. Looks like I was tricked by Debian/Ubuntu's placement: /usr/include/jsoncpp/json/json.h (properly due to naming conflict with just "json")

I looked into it and noticed that jsoncpp could use pkg-config.
I have added it in this commit: https://github.com/blockattack/blockattack-game/commit/cabda704af0acac607bc602f1d00b57b873b0b9c

This is solved in master and will be ready for 2.1.0.
Could you checkout "master" and test it?

**Comment by mstraube** [comment]
Yes it works now. Thank you.

With the latest master I noticed that the man page, .desktop file in icons are installed to `CMAKE_INSTALL_PREFIX/`

I opened a PR for that.
